### PR TITLE
Pin base docker image

### PR DESCRIPTION
**Pull request by Inwerpsel**
* To trigger a new build, latest should also fetch this version, but
* Pinning is better to have control over when a new base is used, as
that can break things.

We need this to get python3 in the builder.

### CODE REVIEW

File: config.default [hunk 1]
```diff
@@ -59,8 +59,8 @@ NAMESPACE=gcr.io
 
 # Upstream image
 BASE_IMAGE=php
-BASE_NAMESPACE=circleci
-BASE_TAG=7.3-node
+BASE_NAMESPACE=cimg
```
Comment (lithrel) [inline config.default:62]: right, circleci has python2 on Debian 10, cimg has python3 on Ubuntu 20.04; cimg seems to be part of "Next gen service images" https://circleci.com/docs/2.0/circleci-images/#next-gen-service-images

Comment (lithrel) [reply config.default:62]: which sounds like a good choice https://circleci.com/blog/announcing-our-next-generation-convenience-images-smaller-faster-more-deterministic/

Comment (Inwerpsel) [reply config.default:62]: Indeed, I'm having a shot at making it work :crossed_fingers:

**Comment by Inwerpsel** [comment]
https://github.com/greenpeace/planet4-circleci/pull/41

**Pull request closed without merging**